packages/desktop/components/NetworkCard.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+    import { localize } from '@core/i18n'
+    import { NetworkConfigRoute, networkConfigRouter } from '@desktop/routers'
+    import { ClickableTile, Text, Icon, FontWeight, TextType, NetworkIcon, NetworkStatusPill } from '@ui'
+    import { Icon as IconEnum } from '@auxiliary/icon'
+    import { truncateString } from '@core/utils'
+    import { NetworkHealth, NetworkId } from '@core/network'
+
+    export let name: string
+    export let address: string
+    export let status: NetworkHealth
+
+    function onTileClick(): void {
+        $networkConfigRouter.goTo(NetworkConfigRoute.ChainInformation)
+    }
+
+    function onQrCodeIconClick(): void {
+        $networkConfigRouter.goTo(NetworkConfigRoute.ChainDepositAddress)
+    }
+</script>
+
+<ClickableTile classes="bg-transparent border border-solid border-gray-200" onClick={onTileClick}>
+    <div class="w-full flex flex-col gap-5">
+        <div class="flex flex-row justify-between items-center">
+            <div class="flex flex-row gap-2 items-center">
+                <NetworkIcon networkId={NetworkId.Shimmer} />
+                <Text type={TextType.h4} fontWeight={FontWeight.semibold}>
+                    {name}
+                </Text>
+            </div>
+            <NetworkStatusPill {status} />
+        </div>
+        <div class="flex flex-row justify-between items-end">
+            <div class="flex flex-col">
+                <Text type={TextType.pre} fontWeight={FontWeight.medium} color="gray-600">
+                    {localize('general.myAddress')}
+                </Text>
+                <Text type={TextType.h4} fontWeight={FontWeight.medium}>
+                    {truncateString(address, 8, 8)}
+                </Text>
```

Comment:
We need to change the font to be `code` font (forgot the name)

Reply:
"My address" should be regular font, then the address itself needs to be `pre` - sorry if I got that mixed up in an earlier review comment :sweat_smile: